www/caddy/src/opnsense/mvc/app/models/OPNsense/Caddy/Caddy.xml

```diff
@@ -85,15 +83,13 @@
                 <Default>10</Default>
                 <MinimumValue>1</MinimumValue>
                 <MaximumValue>3600</MaximumValue>
-                <ValidationMessage>Please enter a valid Grace Period between 1 and 3600 seconds.</ValidationMessage>
```

Comment:
Any particular reason the validation message is being removed here? There are more removed validation messages where there are constraints in place

Reply:
I thought the IntegerField prints a validation message with min and max values:
https://github.com/opnsense/core/blob/706670036de7f314deb17d0be1feff78a809f403/src/opnsense/mvc/app/models/OPNsense/Base/FieldTypes/IntegerField.php#L103

Reply:
No, the validation message just pulls the default of 'Invalid integer value'.

Reply:
Okay then I have to re-add these validation messages, I have interpreted that line wrong.